src/main/java/net/minestom/server/registry/Registry.java

```diff
@@ -261,6 +267,16 @@ public enum Resource {
         }
     }
 
+    public record FeatureFlagEntry(
+            @NotNull NamespaceID namespace,
+            @Nullable Properties custom
+    ) implements Entry {
+
+        public FeatureFlagEntry(String namespace, Properties main, Properties custom) {
```

Comment:
Is there a reason why this constructor has three parameters while the record only takes two?

Reply:
Because main is everywhere needed, Custom also and namespace but not value is present